packages/eslint-plugin/src/rules/no-implied-eval.ts

```diff
@@ -80,6 +80,19 @@ export default util.createRule({
         return true;
       }
 
+      if (
+        symbol &&
+        tsutils.isSymbolFlagSet(
+          symbol,
+          ts.SymbolFlags.FunctionScopedVariable |
+            ts.SymbolFlags.Interface |
+            ts.SymbolFlags.Transient,
+        ) &&
```

Comment:
if you have any good idea to check `symbol.flags === 33554497` please let me know. ✨

Reply:
It might be better instead to just do something similar to what we've got for checking if the symbol is declared locally or not:

https://github.com/typescript-eslint/typescript-eslint/blob/2ada5aff1ef37bc260d7a0eaafe9ff04f8a08fe4/packages/eslint-plugin/src/rules/no-implied-eval.ts#L128-L135

Reply:
@bradzacher Thank you for review. I updated to check if the symbol is declared locally or not :)